src/Platforms/AbstractPlatform.php

```diff
@@ -1988,7 +1986,7 @@ public function prefersIdentityColumns(): bool
      *
      * @return mixed A boolean database value or an array of them.
      */
-    public function convertBooleans($item)
+    public function convertBooleans(mixed $item): mixed
```

Comment:
The return type is `array|int`. What a horrible API.

Reply:
Well, currently you could pass anything to this method and unless it's a boolean or array, the method would just return your argument back to you. The current declaration is accurate, i'm afraid. 🙈